tests-configuration-remoting/src/test/java/org/jboss/hal/testsuite/test/configuration/remoting/HttpConnectorSecurityTest.java

```diff
@@ -48,13 +54,18 @@
 
     @BeforeClass
     public static void beforeClass() throws Exception {
+        client.apply(new AddSocketBinding.Builder(HTTP_CONNECTOR_SECURITY).build());
```

Comment:
I strongly suggest to name socket binding different (but unique name, specific to the test) than the tested connector to avoid confusion.

Reply:
Sorry for this you are absolutely right. I added specific constants for sockets and http listeners.